packages/clay-dropdown/src/ClayDropdownBase.js

```diff
@@ -49,6 +51,28 @@ class ClayDropdownBase extends ClayComponent {
 		this._eventHandler.removeAllListeners();
 	}
 
+	/**
+	 * Align dropdown menu.
+	 * @protected
+	 */
+	_alignDropdown() {
+		if (this.expanded && this._alignElementSelector) {
+			let alignElement = this.element.querySelector(
+				this._alignElementSelector
+			);
+
+			if (alignElement) {
+				let bodyElement = this._getMenuElement();
+
+				this._alignedPosition = Align.align(
+					bodyElement,
+					alignElement,
+					Align.BottomLeft
```

Comment:
hey @carloslancha, can you add our recent change here? `this.preferredAlign`.

We also have to remove `_handleRenderedPortal` since we are no longer using it.

Reply:
Something happened in the rebase of this. I'll resend, thx.